src/main/java/controllers/NotificationController.java

```diff
@@ -18,17 +18,25 @@
 @Controller
 public class NotificationController
 {
+    private UserConfiguration configuration = UserConfiguration.getInstance();
+
     @Autowired
     private SimpMessagingTemplate template;
 
     @RequestMapping(value = "/send-update", method = RequestMethod.POST)
-    @SendTo("/topic/myscores")
     public String handleRequest(HttpServletRequest request, HttpServletResponse response)
     {
-        RequestProcessor processor = new MessengerRequestProcessor();
+        final RequestProcessor processor = new MessengerRequestProcessor();
         final List<Notification> notificationList = new ArrayList<>();
         notificationList.add(processor.processRequest(request));
-        template.convertAndSend("/topic/myscores", notificationList);
+        configuration.getTriggerEndpointConfigurations()
+                .stream()
+                .filter(endpointConf -> endpointConf.getTriggerEndpoint().equals("/app")) //we can pass stuff in
```

Comment:
We don't need to pass any additional information in request to do it. Instead we can just use `request.getRequestURI()` which will provide the data we need. I'm not sure if the trigger endpoints should have any prefix prepended and if so should it be also given in `UserConfiguration` when a user specifies those endpoints. In some scenarios the prefix will need to be removed here before comparison.